docs/getting-started/quickstart-programming.md

```diff
@@ -286,12 +316,11 @@ You should see an output like:
 [info]: Greeting to Mike recorded in the database!
 ```
 
-Notice how DBOS automatically resumes your workflow from where it left off.
-It doesn't re-send the greeting email, but does record the previously-sent greeting in the database.
-This reliability is a core feature of DBOS: workflows always run to completion and each of their operations executes exactly once.
-To learn more about workflows, check out our [tutorial](../tutorials/workflow-tutorial.md) and [explainer](../explanations/how-workflows-work.md).
+If this were not a DBOS `@Workflow` you would expect the app to restart with a "clean slate." There would be one record in the guestbook without matching records in Postgres. But this workflow automatically resumes where it left off and finishes recording the greeting in the database. This reliability is a core feature of DBOS: workflows always continue execution from the last completed step and run completion. To learn more about workflows, check out our [tutorial](../tutorials/workflow-tutorial.md) and [explainer](../explanations/how-workflows-work.md).
+
+Also, note that we start the workflow asynchronously using `startWorkflow`. This returns the response to the caller as soon as the workflow starts, without waiting for the workflow to return. DBOS guarantees that the workflow continues to process to completion. This behavior is preferred when the caller expects a fast response, such as with a [payment webhook](https://www.dbos.dev/blog/open-source-typescript-stripe-processing). To make it synchronous, you can switch `startWorkflow` with `invokeWorkflow`.
```

Comment:
`startWorkflow` is important, but too advanced for this tutorial I think. I'd recommend changing back to `invokeWorkflow` and removing this paragraph.

Reply:
Leaving in, as discussed

Reply:
I'd still make it an infobox to stress that it's optional advanced information

Reply:
done